utility/src/main/java/org/oppia/util/firebase/CrashLogger.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.util.firebase
+
+import java.lang.Exception
+
+/**
+ * Interface that enables exception logging.
+ */
+interface CrashLogger {
+
+  /** @param [exception] : the non fatal exception that is mostly caught in try-catch blocks.
```

Comment:
This KDoc is formatted incorrectly (see https://developer.android.com/kotlin/style-guide#formatting_2), and it's missing a summary.